src/app/monitoring/monitoring.component.ts

```diff
@@ -7,9 +12,55 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MonitoringComponent implements OnInit {
 
-  constructor() { }
+  editors: Editor[];
+  editorSelectorItems: EditorSelectorItem[];
+  selectedEditor: Editor;
+  monitoringEditor: Editor;
+  selectedMedia: Media;
+  jobSuccess = false;
```

Comment:
c'est mieux d'utiliser les observables (BehaviorSubject) pour gérer les valeurs booléennes. En plus tu feras l'économie d'une variable.
```suggestion
  jobState = new BehaviorSubject(false)
  ```
Ensuite this.jobState.next(true ou false en fonction du context)